internal/juju/secrets.go

```diff
@@ -29,8 +29,8 @@ type CreateSecretOutput struct {
 }
 
 type ReadSecretInput struct {
+	SecretId  string
 	ModelName string
-	Name      string
```

Comment:
You're going to have to handle either case. One for a normal read where we have the secretURI and one in the case of import, which will have the name. Checkout the different ImportState methods.

Reply:
Fixed, not Read is more flexible.